nimhapkg/resources/utils/common.nim

```diff
@@ -0,0 +1,23 @@
+
+import parsecfg
+from os import getAppDir
+from strutils import replace
+
+proc loadConf*(modulename: string): Config =
+  ## Load config for the main daemon or a module
+  var fn = ""
+  when defined(dev):
+    if modulename == "":
+      # main daemon
+      fn = getAppDir() & "/config/secret.cfg"
+    else:
+      fn = replace(getAppDir(), "/nimhapkg/mainmodules", "") & "/config/secret.cfg"
+  else:
+    if modulename == "":
+      fn = "/etc/nimha/nimha.cfg"
+    else:
+      fn = "/etc/nimha/" & modulename & ".cfg"
```

Comment:
`/etc/nimha/" & modulename & ".cfg` does not exists. Are you planning to split the *.cfg-file per module or what are your thoughts on this approach? If the file is split into multiple files, it could cause unnecessary work for the user, who needs to open and close a file per module (e.g. when we support +100 plugins, it would take a considerable amount of time).

Reply:
The main reason is that one module should not be able to access credentials stored in a config file for another module. On second thought everything could stay in one config file and then nimha coordinates access (e.g. hands the right config chunk to the right module).

Reply:
Good thought on not sharing credentials, that could be important for future modules. On the other hand, if you install a module, I would expect the user to trust the module code for not accessing other stuff without reason. I think one config file would give the best UX when editing them, but we could simulate that through editing them via the browser. I'm open for both suggestions.

Reply:
I switched back to a single file (at least temporarily) so that the PR can be merged. I'll open more PRs.